StreamingAppDemo/app/src/main/java/com/fortyseven/degrees/streamingapp/predef.kt

```diff
@@ -97,3 +113,26 @@ fun <A, B> Observable<A>.filterMap(f: (A) -> Option<B>): Observable<B> =
         )
     }
 
+fun <A> Observable<Option<A>>.filterOption(): Observable<A> =
+        filterMap(::identity)
+
+
+typealias ObservableSchedule<In, Out> = Schedule<ForObservableK, In, Out>
+
+object ScheduleForMonadDefer : Schedule.Companion.ScheduleFor<ForObservableK> {
+    override fun MM(): Monad<ForObservableK> = ObservableK.monadDefer()
+    fun <In, Out> Schedule<ForObservableK, In, Out>.jittered() =
+        jittered(ObservableK.monadDefer())
+}
+
+/**
+ * DSL entry point to write Schedule's for RxJava without dealing with `Kind` or typeclasses
+ *
+ * fun <A> jitteredRetry(): ObservableSchedule<A, A> =
+ *   RxSchedule {
+ *     spaced<A>(300.milliseconds).jittered() zipRight identity<A>()
+ *   }
+ */
+fun <In, Out> RxSchedule(
```

Comment:
Added a DSL to create an `RxSchedule` for `Observable` without having to pass a typeclass to every method.